.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+
+## Fixes 
+<!-- For e.g Fixes: https://issues.redhat.com/browse/HAC-XXX -->
+
+
+## Description
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+
+## Type of change
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Feature
+- [ ] Bugfix
+- [ ] Code style update (formatting, renaming)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build related changes
+- [ ] Documentation content changes
+- [ ] Other (please describe):
+
+## Screen shots / Gifs for design review 
+<!-- If change affects UI in any way, tag relevant UX people and add screenshots/gifs  -->
+
+
+## How Has This Been Tested?
+<!-- Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration -->
+
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration(s)**:
+
+* Firmware version:
+* Hardware:
+* Toolchain:
+* SDK:
+
+
+## Browser conformance: 
+<!-- To mark tested browsers, use [x] -->
+- [ ] Chrome
+- [ ] Firefox
+- [ ] Safari
+- [ ] Edge
+
+
+## Checklist:
```

Comment:
Is this expected to be deleted by the author or kept in the PR?
If the expectation is to delete, then perhaps make this a comment.

Reply:
Really like a list for PR authors to follow before creating a PR.  Once the PR is created, this may just create clutter in the description.  What if we made this list a comment then it would be visible for authors, but not when viewing the PR?

Also wondering if we can simply each item so it is easier to scan.  So for example, instead of having `My code follows the style guidelines of this project` just shorten it to something like `Code follows style guidelines` or even `Style guidelines were followed` (that way the "action" or most important part of the statement is first.

Reply:
I added this section just to enforce that the author followed certain steps before raising the PR and having it not in comment makes it visible to reviewers as well.

Reply:
I've simplified the steps a bit and added them in a comment as suggested.